tensorflow/java/src/main/java/org/tensorflow/Graph.java

```diff
@@ -77,6 +80,22 @@ public Operation operation(String name) {
   }
 
   /**
+   * Returns all operations that exist in the Graph.
+   */
+  public List<Operation> getOperations() {
```

Comment:
Let's call this `operations()` instead of `getOperations()` - keeping with the convention of avoiding the `get` prefix on other methods in this package.